modules/binary-authorization/main.tf

```diff
@@ -0,0 +1,42 @@
+resource "google_binary_authorization_attestor" "attestor" {
```

Comment:
Missing copyright header

Reply:
Added the copyright header for all files and a README file showing how to use the module